packages/sdk/src/client.ts

```diff
@@ -728,6 +730,57 @@ export class Client
         return { streamId: streamId, prevMiniblockHash: streamView.prevMiniblockHash }
     }
 
+    async createSpaceMediaStream(
+        spaceId: string | undefined,
+        chunkCount: number,
+        streamSettings?: PlainMessage<StreamSettings>,
+    ): Promise<{ streamId: string; prevMiniblockHash: Uint8Array }> {
+        assert(spaceId !== undefined, 'userStreamId must be set')
+        assert(this.userStreamId !== undefined, 'userStreamId must be set')
+        assert(isSpaceStreamId(spaceId), 'spaceId must be a valid streamId')
+
+        const streamId = makeMediaStreamIdFromSpaceId(spaceId)
+        const streamIdBytes = streamIdAsBytes(streamId)
+        const spaceIdBytes = streamIdAsBytes(spaceId)
+
+        this.logCall('createSpaceMediaStream', streamId)
+
+        const inceptionEvent = await makeEvent(
+            this.signerContext,
+            make_MediaPayload_Inception({
+                streamId: streamIdBytes,
+                spaceId: spaceIdBytes,
+                channelId: spaceIdBytes,
```

Comment:
I don’t think we should set the channel id to the space id. This seems weird to me.

Reply:
agreed. My first cut was to set channelId to `new UintArray(0)` and `new UIntArray(32)` (tried both). But we have strict rule checks on the river node that rejects attempt to create a media stream with an invalid channelid, not tied to any channelId, and without a membership to the associated `channel` stream. 
Tried to disentangled to strict coupling between media and channel, but when i debug the code, I realize that these are good checks to prevent unintended 'mistakes'. e.g. stream creator needs to be a member of the stream already before a media stream associated with that stream can be created. 

Since `channel` is a concept for storing contents, my rationale for setting the 'channelId' to the spaceId is that all the channel rules and logic should apply. Down the road, this `space channel` can be an  `admin channel` for other contents, like bot communication, and what not.

If not, then I'll need your help to understand what rules I can break / change between the space channel, and a regular channel.